core/templates/dev/head/no-script.template.html

```diff
@@ -0,0 +1,19 @@
+<noscript>
+  <div class="oppia-page-cards-container">
+    <div class="md-default-theme oppia-page-card oppia-long-text">
+      <!-- Note to developers: We replicate the translated text inline because, without JavaScript enabled, the translation engine doesn't kick in.-->
+      <h2>
+        <span translate="I18N_SPLASH_JAVASCRIPT_ERROR_TITLE">We Need JavaScript in Your Browser</span>
+        <i class="material-icons">&#xE811;</i>
+      </h2>
+      <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION"
+         translate-values="{hrefUrl: 'http://www.enable-javascript.com/'}">
+        Oppia is a free, open-source learning platform full of interactive activities called
+        'explorations'. Sadly, Oppia requires JavaScript to be enabled in your web browser in
+        order to function properly and your web browser has JavaScript disabled. If you need
+        help enabling JavaScript, <a href="http://www.enable-javascript.com">click here.</a>
+      </p>
+      <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_THANKS">Thank you.</p>
+    </div>
+  </div>
+</noscript>
```

Comment:
Maybe place this file into `base-components`?

Reply:
Done